Index: app/build.gradle
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/build.gradle	(revision ed5afb0708bcf9d06e8c5d21a221c3253f7d6984)
+++ app/build.gradle	(revision )
@@ -42,4 +42,5 @@
     implementation 'com.android.support:support-v13:27.0.2'
     implementation 'com.trello.rxlifecycle2:rxlifecycle-android-lifecycle:2.2.1'
     implementation files('libs/open_sdk_r5990_lite.jar')
+    implementation 'com.squareup.okhttp3:mockwebserver:3.10.0'
 }
Index: app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDetailedFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDetailedFragment.java	(revision ed5afb0708bcf9d06e8c5d21a221c3253f7d6984)
+++ app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDetailedFragment.java	(revision )
@@ -38,13 +38,14 @@
     private ArrayList<GameInfoPicModel> picList;
     private ArrayList<GameInfoThemeModel> themeList;
     private static final String TAG = "GameInfoDetailedFragmen";
+
     @Nullable
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
         View v = inflater.inflate(R.layout.fragment_gameinfo_detailed, container, false);
         initViews(v);
         initData();
-        Log.e(TAG, "onCreateView: "+"a" );
+        Log.e(TAG, "onCreateView: " + "a");
         picRv.setLayoutManager(new LinearLayoutManager(getActivity(), LinearLayoutManager.HORIZONTAL, false));
         picAdapter = new GameInfoPicAdapter(getActivity(), picList);
         picRv.setAdapter(picAdapter);
@@ -58,6 +59,17 @@
 
     private void initData() {
         picList = new ArrayList<>();
+        // TODO: 2018/3/18 tset
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854233&di=cf3bcab5c0e3ee4deb851b2b26c3b055&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F5ab5c9ea15ce36d3f7ff9d2136f33a87e950b123.jpg "));
+        picList.add(new GameInfoPicModel("https://ss3.baidu.com/9fo3dSag_xI4khGko9WTAnF6hhy/image/h%3D300/sign=ea376b0a7e0e0cf3bff748fb3a46f23d/adaf2edda3cc7cd9f775f2c13501213fb80e914a.jpg                                                                                        "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854231&di=fa395a127c118f075241fec30752d4cc&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Ffaf2b2119313b07e824d771800d7912397dd8c29.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385264791&di=2181b33ec97e75a990820f7849ebf463&imgtype=0&src=http%3A%2F%2Fb.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F18d8bc3eb13533fa434d443ea4d3fd1f41345b23.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648469&di=2646274010a3dd82c841dd9e70de8242&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fb3119313b07eca807eff4cff9d2397dda144832b.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648469&di=f2393359939578f829fbf9405fae3a48&imgtype=0&src=http%3A%2F%2Fe.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fbd3eb13533fa828bc264e8fbf11f4134970a5a66.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648469&di=bd55edfec49cae8e68874be0356a9974&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F8718367adab44aedec01cafcbf1c8701a18bfb5f.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648468&di=ecadd31efb0e7ed417dad0524a3209dd&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F4b90f603738da9778360f233bc51f8198618e35f.jpg "));
+        picList.add(new GameInfoPicModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854232&di=529a7b60aaa03551a757d58c34036772&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fcc11728b4710b912c87f7fedcffdfc0392452262.jpg "));
+
         themeList = new ArrayList<>();
 
     }
Index: app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDiscussFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDiscussFragment.java	(revision ed5afb0708bcf9d06e8c5d21a221c3253f7d6984)
+++ app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoDiscussFragment.java	(revision )
@@ -46,6 +46,33 @@
 
     private void initData() {
         list = new ArrayList<>();
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648468&di=47cec5dbb8bf70d4710f09e58de4ee8d&imgtype=0&src=http%3A%2F%2Fg.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fae51f3deb48f8c54d2a0b13736292df5e0fe7fb2.jpg","2017-12-18","a","a","1900","212311313","huawei mate8","liu","2",1,12,10,92));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648468&di=f185ab624c8030d4995f9eaa7bcdfb26&imgtype=0&src=http%3A%2F%2Fh.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F7dd98d1001e939010e71f92677ec54e736d196b2.jpg","2017-12-18","b","aa","1600","212311313","huawei mate9","liu","2",2,122,120,91));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648468&di=a641414ce2a95374cc455ae258d01d20&imgtype=0&src=http%3A%2F%2Fh.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F37d3d539b6003af335c8a878392ac65c1038b6b3.jpg","2017-12-18","c","af","15600","212311313","huawei mate10","liu","2",3,142,110,29));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648467&di=646b4aeca7a50ebb86c3d0eb4eda6cf2&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F9a504fc2d5628535df92a6399cef76c6a7ef63b3.jpg","2017-12-18","d","af","1260","212311313","huawei mate11","liu","2",4,142,120,91));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648467&di=a567fc4f9422e7b21ec21f5940f77ea8&imgtype=0&src=http%3A%2F%2Fb.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F96dda144ad34598293c4918a00f431adcbef845f.jpg","2017-12-18","e","afa","12960","212311313","huawei mate12","liu","2",5,42,110,29));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648467&di=a567fc4f9422e7b21ec21f5940f77ea8&imgtype=0&src=http%3A%2F%2Fb.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F96dda144ad34598293c4918a00f431adcbef845f.jpg","2017-12-18","f","aa","1260","212311313","huawei mate123","liu","2",6,132,120,19));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648467&di=95b041cbb104b233cb717e0a2a7cb491&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fd52a2834349b033b53c188c219ce36d3d539bd58.jpg","2017-12-18","g","af","1800","212311313","huawei mate15","liu","2",7,124,110,92));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648466&di=ea738255af2208d8b3db3be9a42ac2ca&imgtype=0&src=http%3A%2F%2Ff.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fb2de9c82d158ccbf31fdc02215d8bc3eb1354160.jpg","2017-12-18","h","aav","1290","212311313","huawei mate8","liu","2",8,126,120,91));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648466&di=9b8c1181698293daa0acc612eb83f148&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F4b90f603738da97782dbf333bc51f8198618e336.jpg","2017-12-18","i","va","1920","212311313","huawei mate7","liu","2",9,127,110,92));
+        list.add(new GameInfoDiscussModel("https://ss2.baidu.com/-vo3dSag_xI4khGko9WTAnF6hhy/image/h%3D300/sign=b3d3dd3ef1faaf519be387bfbc5594ed/738b4710b912c8fcbe0683d5f0039245d6882134.jpg","2017-12-18","a","a","1900","219+5+651313","xiaomi mate8","lu","我是小米",10,1532,130,94));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385264789&di=2498c3a856e2ecf0463461fb1167acff&imgtype=0&src=http%3A%2F%2Ff.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F2fdda3cc7cd98d105c9f28202d3fb80e7bec9053.jpg","2017-12-18","k","vaa","1600","212311313","huawei mate6","liu","2",11,182,120,1));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385264788&di=27d51342a189ebfc7b9a6d5c01cc5125&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F8b82b9014a90f603e0fbcf573512b31bb051ed53.jpg","2017-12-18","l","aa","1600","212311313","huawei mate4","liu","2",12,172,110,92));
+        list.add(new GameInfoDiscussModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385648468&di=a777dbf4a04a2649ddded6d6d630e0b2&imgtype=0&src=http%3A%2F%2Fe.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F5882b2b7d0a20cf415e38ac77a094b36acaf99b2.jpg","2017-12-18","b","da","1860","212311313","huawei mate6","liu","2",13,192,120,91));
+
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
+        
     }
 
     private void initViews(View v) {
Index: app/src/main/java/com/demo/yetote/cubegame/fragment/BoonFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/demo/yetote/cubegame/fragment/BoonFragment.java	(revision ed5afb0708bcf9d06e8c5d21a221c3253f7d6984)
+++ app/src/main/java/com/demo/yetote/cubegame/fragment/BoonFragment.java	(revision )
@@ -49,5 +49,28 @@
 
     private void initData() {
         list = new ArrayList<>();
+        // TODO: 2018/3/18 test
+        list.add(new BoonModel("a", "free", "https://ss1.baidu.com/9vo3dSag_xI4khGko9WTAnF6hhy/image/h%3D300/sign=e79fdbb1067b020813c939e152d8f25f/14ce36d3d539b600b36afca3e550352ac65cb77a.jpg", "''asfgvagsdfasfasf", "%50", "￥85", "￥170", 1, 176));
+        list.add(new BoonModel("b", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854240&di=a87480aa3a343e8666c22f855d5bd524&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fd833c895d143ad4b3ae286d88e025aafa50f06de.jpg", "''asfgvagsdfasfasf", "%50", "￥85", "￥170", 1, 76));
+        list.add(new BoonModel("c", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854239&di=d0a31ef22e8cc3fea4c6c196aeeaad78&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F72f082025aafa40f8197e0cca764034f78f01949.jpg", "''asfgvagsdfasfasf", "%70", "￥85", "￥170", 2, 76));
+        list.add(new BoonModel("d", "count", "https://ss3.baidu.com/9fo3dSag_xI4khGko9WTAnF6hhy/image/h%3D300/sign=e7ea61e75dda81cb51e685cd6267d0a4/4bed2e738bd4b31ccd851da88bd6277f9e2ff86c.jpg", "''asfgvagsdfasfasf", "%50", "￥85", "￥170", 3, 156));
+        list.add(new BoonModel("e", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854239&di=25eb520282df643a2cebf20def89e9a7&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F0dd7912397dda144a5db01a2beb7d0a20df486cb.jpg", "''asfgvagsdfasfasf", "%50", "￥525", "￥240", 4, 87));
+        list.add(new BoonModel("f", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854239&di=25eb520282df643a2cebf20def89e9a7&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F0dd7912397dda144a5db01a2beb7d0a20df486cb.jpg", "''asfgvagsdfasfasf", "%80", "￥35", "￥150", 5, 76));
+        list.add(new BoonModel("g", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854239&di=cf475721d95f754b733a7d72c3a35816&imgtype=0&src=http%3A%2F%2Fe.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F359b033b5bb5c9eaa91bb9f0d939b6003af3b33d.jpg", "''asfgvagsdfasfasf", "%52", "￥755", "￥150", 6, 16));
+        list.add(new BoonModel("h", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854239&di=be61b728a9a40a1321591505309ace82&imgtype=0&src=http%3A%2F%2Fc.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F962bd40735fae6cd09ccfb7903b30f2442a70fa9.jpg", "''asfgvagsdfasfasf", "%42", "￥5", "￥150", 7, 18));
+        list.add(new BoonModel("i", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854238&di=4c9a01ca62ba495935e4e995d6557451&imgtype=0&src=http%3A%2F%2Fe.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F3801213fb80e7bec00e429f3232eb9389b506be1.jpg", "''asfgvagsdfasfasf", "%25", "￥65", "￥140", 8, 15));
+        list.add(new BoonModel("j", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854238&di=09c307ecd99c3d5dff335df49446971b&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fe850352ac65c10384bfac6d1be119313b07e8918.jpg", "''asfgvagsdfasfasf", "%86", "￥25", "￥17", 9, 15));
+        list.add(new BoonModel("k", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854238&di=cdd8d3382586dbfaeee674bb40fc7ab6&imgtype=0&src=http%3A%2F%2Fh.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F4afbfbedab64034faff6e516a3c379310a551de2.jpg", "''asfgvagsdfasfasf", "%70", "￥80", "￥170", 10, 56));
+        list.add(new BoonModel("l", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854237&di=c6b54266880186644225a10fe78a1c37&imgtype=0&src=http%3A%2F%2Ff.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F9f2f070828381f305c2ce1b9a5014c086e06f0ed.jpg", "''asfgvagsdfasfasf", "%80", "￥85", "￥190", 11, 16));
+        list.add(new BoonModel("m", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854237&di=36c5ab3e428215cb157db298682b8235&imgtype=0&src=http%3A%2F%2Fg.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F8435e5dde71190ef500dffcfc21b9d16fdfa602f.jpg", "''asfgvagsdfasfasf", "%53", "￥88", "￥17", 12, 16));
+        list.add(new BoonModel("n", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854236&di=09db46b56156478d7305ae24bee88ed8&imgtype=0&src=http%3A%2F%2Ff.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fb03533fa828ba61ee51aa53f4d34970a304e592d.jpg", "''asfgvagsdfasfasf", "%50", "￥87", "￥13", 13, 176));
+        list.add(new BoonModel("o", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854236&di=dcfa517e4daaee2337b9cc761364921a&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F6f061d950a7b02080870a9f26ed9f2d3572cc86c.jpg", "''asfgvagsdfasfasf", "%40", "￥95", "￥140", 14, 176));
+        list.add(new BoonModel("p", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854236&di=f20702f79e03ada56cafe34db2c03358&imgtype=0&src=http%3A%2F%2Fb.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2F203fb80e7bec54e77fe8040fb5389b504fc26a9e.jpg", "''asfgvagsdfasfasf", "%45", "￥75", "￥173", 15, 156));
+        list.add(new BoonModel("q", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854235&di=80438a31abacfcd86825700322b7ffec&imgtype=0&src=http%3A%2F%2Fg.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fcaef76094b36acaf7853b7ed70d98d1001e99c6b.jpg", "''asfgvagsdfasfasf", "%7", "￥86", "￥170", 16, 15));
+        list.add(new BoonModel("r", "count", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854234&di=53c0f23553c6c3a2462241fb1baf57b3&imgtype=0&src=http%3A%2F%2Ff.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Ff2deb48f8c5494ee850a290821f5e0fe99257ef2.jpg", "''asfgvagsdfasfasf", "%80", "￥95", "￥180", 17, 156));
+        list.add(new BoonModel("s", "count", "https://ss0.baidu.com/7Po3dSag_xI4khGko9WTAnF6hhy/image/h%3D300/sign=316a9f2b24381f3081198ba999014c67/6159252dd42a28349bde9b1857b5c9ea15cebfad.jpg", "''asfgvagsdfasfasf", "%50", "￥85", "￥170", 18, 15));
+        list.add(new BoonModel("t", "free", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521384854233&di=a03fc3bcc3b6da9fa6e0168ea44136ce&imgtype=0&src=http%3A%2F%2Fa.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fcc11728b4710b912881abfedcffdfc03934522cd.jpg", "''asfgvagsdfasfasf", "%50", "￥85", "￥170", 19, 176));
     }
+
+
 }
Index: app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoForumFragment.java
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoForumFragment.java	(revision ed5afb0708bcf9d06e8c5d21a221c3253f7d6984)
+++ app/src/main/java/com/demo/yetote/cubegame/fragment/GameInfoForumFragment.java	(revision )
@@ -10,7 +10,6 @@
 import android.view.ViewGroup;
 import android.widget.AdapterView;
 import android.widget.ArrayAdapter;
-import android.widget.SimpleAdapter;
 import android.widget.Spinner;
 
 import com.demo.yetote.cubegame.R;
@@ -61,6 +60,16 @@
 
     private void initData() {
         list = new ArrayList<>();
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521385264788&di=4fd841145339e233dde352f3dd008e76&imgtype=0&src=http%3A%2F%2Fd.hiphotos.baidu.com%2Fimage%2Fpic%2Fitem%2Fb151f8198618367a29fb44f322738bd4b31ce5aa.jpg", "今日开启测试", "2018-01-12", "ceshikaiqile", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626908&di=eeec5f969634c435409a3c31fd48f412&imgtype=0&src=http%3A%2F%2Fpic.yesky.com%2FuploadImages%2F2015%2F215%2F45%2F04L5VRR21C5W.jpg", "看美女啊", 12712, 112, 2, 1));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626908&di=eeec5f969634c435409a3c31fd48f412&imgtype=0&src=http%3A%2F%2Fpic.yesky.com%2FuploadImages%2F2015%2F215%2F45%2F04L5VRR21C5W.jpg", "今日开启测试", "2018-10-12", "666", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626896&di=84d586d19891bd5adecf4b13f935e714&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F023b5bb5c9ea15cec98a9b35bd003af33a87b2a2.jpg", "看美女啊1", 12812, 12, 24, 2));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626907&di=d680b5474da2e5e3f2b754439fc16c93&imgtype=0&src=http%3A%2F%2Fimg1.xiazaizhijia.com%2Fwalls%2F20151230%2F1440x900_6a3ec17c19e5064.jpg", "今日开启测试", "2018-11-12", "777", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626896&di=a589e985f96901e820eb8d9d0a47eba1&imgtype=0&src=http%3A%2F%2Fpic47.nipic.com%2F20140902%2F8607306_233416291000_2.jpg", "看美女啊2", 1212, 112, 224, 3));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626906&di=f9d826545a0b4eb3848542ca049a4e0f&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2Fe1fe9925bc315c600dce09d386b1cb13495477b6.jpg", "今日开启测试", "2018-01-12", "98", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626926&di=4a554de5e8aa35b3cb7f8dc1c116753f&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F77094b36acaf2edd47c3f4d4861001e9390193d9.jpg", "看美女啊3", 12712, 172, 224, 4));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626904&di=6d02616046f66c505dbcf553484c3cea&imgtype=0&src=http%3A%2F%2Fdynamic-image.yesky.com%2F740x-%2FuploadImages%2F2015%2F131%2F33%2FD472XQ25C7H2.jpg ", "今日开启测试", "2018-12-12", "888", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521389154887&di=321669bd0dea6fa072ed5f7eb08c935e&imgtype=jpg&src=http%3A%2F%2Fimg0.imgtn.bdimg.com%2Fit%2Fu%3D2256946865%2C212613265%26fm%3D214%26gp%3D0.jpg", "看美女啊4", 12162, 132, 20, 5));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626902&di=e64698cb1af3b4efa945e73c941a0e66&imgtype=0&src=http%3A%2F%2Fimage.tianjimedia.com%2FuploadImages%2F2015%2F198%2F38%2F94P7DLV3N9UG.jpg", "今日开启测试", "2018-11-12", "895", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626925&di=0f986d6413f70dd942ea7259c314fa6c&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F060828381f30e924ae26530b47086e061d95f70e.jpg", "看美女啊5", 12512, 124, 12, 6));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626900&di=cab4f0ef3d1201236fc2d3e14a069552&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F241f95cad1c8a786ec11f5336c09c93d70cf50d7.jpg", "今日开启测试", "2018-01-12", "856", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626925&di=37d05c0739951eaf0a93b1c7dd06fe5f&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2Faa18972bd40735faa97ff38495510fb30f24087b.jpg", "看美女啊6", 12124, 172, 24, 7));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626899&di=0ec8e7a254fa83d7709cabc751e18ffa&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F3ac79f3df8dcd10088b14b1c798b4710b9122fd9.jpg", "今日开启测试", "2018-10-12", "我看美女", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626925&di=3a5063599781a33965ddff61581d4180&imgtype=0&src=http%3A%2F%2Fimage.tianjimedia.com%2FuploadImages%2F2014%2F266%2F40%2F6E0VK7L4C6O6_1000x500.jpg", "看美女啊7", 14212, 182, 201, 8));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626904&di=f905c9e14ace8de07bc1584186bd945d&imgtype=0&src=http%3A%2F%2Fpic.yesky.com%2FuploadImages%2F2015%2F199%2F11%2FISYG68AU213I.jpg ", "今日开启测试", "2018-01-12", "美女", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626924&di=3afa5c6073b7b509c37b375e45227626&imgtype=0&src=http%3A%2F%2Fimg05.tooopen.com%2Fimages%2F20150929%2Ftooopen_sy_143927879853.jpg", "看美女啊8", 12612, 182, 242,9));
+        list.add(new GameInfoForumModel("https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626908&di=6f2129fa40a63b677bb3d981962ae375&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimage%2Fc0%253Dshijue1%252C0%252C0%252C294%252C40%2Fsign%3D72e134ec88025aafc73f76889384c111%2Fa50f4bfbfbedab64a0bd3fecfd36afc378311eca.jpg", "今日开启测试", "2018-01-15", "666", "https://timgsa.baidu.com/timg?image&quality=80&size=b9999_10000&sec=1521386626923&di=75003c37a4ea3ada6ad49a549dfc84fe&imgtype=0&src=http%3A%2F%2Fimgsrc.baidu.com%2Fimgad%2Fpic%2Fitem%2F3bf33a87e950352a0c1c9a355843fbf2b2118b8a.jpg", "看美女啊9", 12142, 18, 82, 10));
     }
 
     @Override
